### DISCUSSION TITLE
How to have a title, but not display it?

### DISCUSSION THREAD

**Discussion by dense-set** [quarto-dev/quarto-cli#6377] [Q&A]
Hi. In making a webpage, the `title: abc` option in the `.qmd` sets two things in the generated HTML:
1. `<title>abc</title>` in the `<head>`; and,
2. `<h1 class="title">abc</h1>` somewhere inside the `<body>`.

What can I do to ensure that the second thing is not included in the HTML?

**Comment by mcanouil** [comment]
Don't use `title` and use only `pagetitle`, see <https://quarto.org/docs/reference/formats/html.html>.

**Reply by dhimmel** [reply]
Nice. I'd like to go one step further and use `pagetitle` instead of `title` and perhaps `description-meta` instead of `description` when creating a [listing](https://quarto.org/docs/websites/website-listings.html#listing-fields), but it looks like listings only allow referencing predetermined allowed fields.

Actually, it kinda works in grid view but shows key-value pairs:

![image](https://github.com/user-attachments/assets/de719c7a-7caf-48ae-be25-8d79849a4ea2)

**Reply by mcanouil** [reply]
@dhimmel use custom listings and the `include-*` YAML options to do whatever you want to do.
